docs/reference/api/events/_index.mdx

```diff
@@ -2,9 +2,9 @@
 title: 'Events Overview'
 ---
 
-Nitric's events service and API provides an intuitive approach to asynchronous messaging allowing services, functions and more to communicate via topics with event messages. While synchronous request/response communication is quite common, events provide a more reliable mechanism for communication in distributed and serverless systems, with the added benefit that services can be decoupled from each other through a [publish/subscribe](https://en.wikipedia.org/wiki/Publish%E2%80%93subscribe_pattern) relationship.
+Events are a form of asynchronous messaging which enable functions and other stack resources to communicate with one another. Nitric Events follow a [publish/subscribe](https://en.wikipedia.org/wiki/Publish%E2%80%93subscribe_pattern) model, where messages are published to Topics by Publishers and forwarded to Subscribers. While synchronous request/response communication is quite common, events assist in decoupling services from each other and provide mechanisms that improve fault tolerance. These benefits are important in the creation of distributed and serverless systems, where functions and services are often stateless and ephemral.
 
```

Comment:
There are other benefits of pub/sub architectures, temporal decoupling, load leveling, efficient resource use, etc., but this is probably not the righ place to go into all this.